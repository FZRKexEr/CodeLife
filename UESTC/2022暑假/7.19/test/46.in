10
fxhvvpygdnjeyokwbkhlkosvdzgzrueduyhommisohkskjrarvmpejyjfdbhmfituzshdeocwjyijbdhzidmcbklhtqpijyrebyk
4
pnwysqchginypryveybjsqfstfbwmidsdawxdqtjesbkchpghjwaaffioefulpinkixdajxxpfsbttyhpplpqrcdiamyenijqqnp
7
teqexsjwokhstzofvzanvafogysxpqzmciehsysgswmrpsumbkfeuhvsvuiuylkaqapxrgpbuuobttudydqtmpkjfzreixuymbih
9
ihpiysvggajvazxlukxpuvcanurbwtdddfdlyoqkyksjuyyqiymymqowxyyphstiopzmqsfaujlaneemjwulsldlvmbdhitpiils
6
lnyafzugeepbboyclylsgqcsdvlsaszbqlfcqzsrbtuutkynmakvlxxcqwnygzevgvcxmronnyndudarpbnrouyxmjzhqdquuojt
5
qjvfgsvcqqqvzocwjakcecrddppfpmxdcthxyiaeohyoqnlkiklicgbrqkotophkewsdxkdidtzhmgttbhkrzhlqzgnvesaspmbh
2
lytnhjrlovmfdlhgzbrqvuhwwmctglluujqvssnqdespooaaudaecazgrmvbzolhbcoagutbrdwztfdzkajplwmtntfnpherdbiu
2
oxgbwdyeifcesfubpvwchoywmcqncmibndceubudnzbewhadvfwqiybteutlrlnpzrabkucrmatelqhkymucqdoneizidnwqzlmy
10
avrqcptldzoiuhvexdfqsplapzdltytxduukswjtcpyfrirlviktmjjsftsonnenkqiaqhslqeodcqwqtoteyhnubgsyuulvjndw
1
ucgxmiborrjesibwexfbchzyswyxarwiscbprwzsempdbkzvlnxwjcaswutkpvoqabajmzfeokehrdzkcqbpoergfutpernifuik
7
