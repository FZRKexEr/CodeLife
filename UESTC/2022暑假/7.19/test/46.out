33554432
16384
2048
65536
1048576
65980984
65980984
1024
882499718
16384
